### PR TITLE
Angelo datastreams bode

### PR DESCRIPTION
**Pull request by cbode**
Dashboard specific datastreams created.  Python code can be generalized to all datastreams, though it is not optimized.

### CODE REVIEW

**Comment by jscottsf** [comment]
In some datastreams, the datastream_id under the datapoints_config is string. It should be numeric.

**Comment by jscottsf** [comment]
In a lot of datastreams, the field "attributes" is present but empty. Do we want to require that this object contain at least one field when present. Can this be done with JSON schema?

**Comment by jscottsf** [comment]
I started a standard .gitignore that I use for all other repos. We may at some point want to merge/optimize the .gitignore here with the standard one I use. Just to avoid having so many flavors. Nothing actionable for now. Maybe we add this as a GitHub issue/enhancement or task on Trello.

**Comment by cbode** [comment]
I vote to use your standard .gitignore.  The only relevant part of mine was the ignoring of ".pw" files.

**Comment by cbode** [comment]
All issues have been addressed.  Request merge upon review.

File: examples/vocabularies/ds-interval.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "_id": "ds-time-aggregate",
+  "_id": "ds-interval",
   "created_at": "2017-01-31T09:30:00Z",
   "description": "TBD",
-  "label": "Time Aggregate",
+  "label": "Time Interval",
```
Comment (jscottsf) [inline examples/vocabularies/ds-interval.json]: Label should be "Interval", not "Time Interval".

**Pull request merged**